### ISSUE TITLE
To-show patterns

### ISSUE THREAD

**Issue by realAbstractionist** [TheKeyboardInvaders/Project-Life#1]
We gotta create a folder including our cool presets to show the audience. Stuff like Gossling Gun and other.

**Comment by Heidric** [comment]
Yeah, also we can create a menu to load them without the need to go through "Load simulation" dialog.
